storage/AggregationOperationState.hpp

```diff
@@ -156,6 +152,29 @@ class AggregationOperationState {
       const CatalogDatabaseLite &database);
 
   /**
+   * @brief Get the number of partitions to be used for initializing the
+   *        aggregation.
+   *
+   * @return The number of partitions to be used for initializing the aggregation.
+   **/
+  std::size_t getNumInitializationPartitions() const;
+
+  /**
+   * @brief Get the number of partitions to be used for finalizing the
+   *        aggregation.
+   *
+   * @return The number of partitions to be used for finalizing the aggregation.
+   **/
+  std::size_t getNumFinalizationPartitions() const;
```

Comment:
Currently initialization is for `memset` the storage memory. It is much faster than finalization so we expect to have a smaller number of initialization partitions than finalization partitions if the storage memory is not large.

E.g. suppose the vector table has 1 million entries of `int` keys and `std::int64_t` states -- amount to 12MB storage memory. Then we may want 80 partitions to finalize the table, but would not create 80 work orders to initialize the 12MB memory.